manual/manual/tutorials/moduleexamples.etex

```diff
@@ -67,43 +67,43 @@ identifiers present in the current scope, potentially leading
 to confusing errors:
 
 \begin{caml_example}{toplevel}
-  let empty = []
-  open PrioQueue;;
-  let x = 1 :: empty [@@expect error];;
+let empty = []
+open PrioQueue;;
+let x = 1 :: empty [@@expect error];;
 \end{caml_example}
 
 
 A partial solution to this conundrum is to open modules locally,
 making the components of the module available only in the
 concerned expression. This can also make the code easier to read
--- the open statement is closer to where it is used-- and to refactor
+-- the open statement is closer to where it is used -- and to refactor
 -- the code fragment is more self-contained.
```

Comment:
If the parenthetical dashes were replaced, I would prefer commas to parentheses since the propositions are an important part of the flow of the sentence.

Reply:
I have chosen:

This can also make the code both easier to read (since the open statement is closer to where it is used) and easier to refactor (since the code fragment is more self-contained).